generator/Cargo.toml

```diff
@@ -5,13 +5,15 @@ authors = ["Maik Klein <maikklein@googlemail.com>"]
 edition = "2018"
 
 [dependencies]
-vk-parse = { version = "0.5.0", features = ["vkxml-convert"] }
-vkxml = "0.3"
-nom = "6.0"
 heck = "0.3"
-proc-macro2 = "1.0"
 itertools = "0.10"
+nom = "6.0"
+once_cell = "1.7"
+proc-macro2 = "1.0"
 quote = "1.0"
+regex = "1.4"
+vk-parse = { git = "https://github.com/krolli/vk-parse", features = ["vkxml-convert"] }
```

Comment:
Do you know the status for a new release of vk-parse? Otherwise we should target a specific commit here.

Reply:
@krolli has been actively updating his repo over the past days, and the CI seems to succeed again. Best to ask him directly how soon a crate release is possible :)

We can also tag a specific commit, yes.

Reply:
I've explicitly specified the latest master commit now, at https://github.com/krolli/vk-parse/commit/a133cbb330deeb5c8cea63574cb309075e43a231.